src/utils/getLibrary.ts

```diff
@@ -0,0 +1,5 @@
+import { Web3Provider } from '@ethersproject/providers'
```

Comment:
This function should instantiate a web3 instance according to the doc https://github.com/NoahZinsmeister/web3-react/tree/v6/docs#props , not just returning the provider. 

If we do so, the `useWeb3` hook become obsolete, because we can retrieve the web3 instance from the `useWeb3React` hook

Reply:
I updated this part because if we change, we should update the part used `useWeb3` hook